rasa/train.py

```diff
@@ -158,18 +159,19 @@ def train(
         )
 
     old_model = model.get_latest_model(output_path)
-    retrain_core, retrain_nlu = model.should_retrain(
+    retrain_core, retrain_nlu, retrain_nlg = model.should_retrain(
```

Comment:
can't we skip that in case we have a `force_training`?

Reply:
also the `new_fingerprint` initialisation should go down to line 161 since this is the first time it's actually used.